.github/workflows/automated-translations.yml

```diff
@@ -71,7 +72,7 @@ jobs:
           MSG_MINIMAL: 'true'
 
       - name: Notify Slack for fails
-        if: ${{ failure() }}
+        if: ${{ failure() || steps.sync.outputs.success != 'true' }}
```

Comment:
Now notifies for "soft" failures in case something unexpected happens but the script exits cleanly.